src/tactic/transport_2.lean

```diff
@@ -0,0 +1,141 @@
+/-
```

Comment:
For now this file is called `transport_2.lean`, because there is already an existing `transport.lean` used by `to_additive`. I'd propose renaming that `transport.lean` to `transform_decl.lean` or similar.

Reply:
I've now moved it.